packages/expo-cli/src/commands/publish.ts

```diff
@@ -147,61 +110,54 @@ export async function action(projectDir: string, options: Options = {}) {
     simpleSpinner.start();
   }
 
-  let result;
-  try {
-    result = await Project.publishAsync(projectDir, {
-      releaseChannel: options.releaseChannel,
-    });
+  const result = await Project.publishAsync(projectDir, {
+    releaseChannel: options.releaseChannel,
+  });
 
-    let url = result.url;
+  let url = result.url;
 
-    if (options.quiet) {
-      simpleSpinner.stop();
-    }
+  if (options.quiet) {
+    simpleSpinner.stop();
+  }
 
-    log('Publish complete');
-    log.newLine();
-
-    let exampleManifestUrl = getExampleManifestUrl(url, exp.sdkVersion);
-    if (exampleManifestUrl) {
-      log(
-        `The manifest URL is: ${terminalLink(url, exampleManifestUrl)}. ${terminalLink(
-          'Learn more.',
-          'https://expo.fyi/manifest-url'
-        )}`
-      );
-    } else {
-      log(
-        `The manifest URL is: ${terminalLink(url, url)}. ${terminalLink(
-          'Learn more.',
-          'https://expo.fyi/manifest-url'
-        )}`
-      );
-    }
+  log('Publish complete');
+  log.newLine();
 
-    if (target === 'managed') {
```

Comment:
Again I think this should have at least one test before creating such a hefty change. Because it's somewhat of a long running task it needs to be continuously tested.